### DISCUSSION TITLE
Switching store views in Magento 2

### DISCUSSION THREAD

**Discussion by jellesiderius** [bjarn/jale#80] [Ideas]
It would be nice to be able to change between store views easily. Maybe through a .conf file within the project folder?

**Comment by bjarn** [comment]
You can already set a MAGE_RUN_CODE in the `~/.jale/sites/<project>.conf`, however a Jale config which can be used from within a project is also on the roadmap: https://github.com/bjarn/jale/issues/15

Is that what you were looking for?

**Reply by jellesiderius** [reply]
Ah! That would resolve this indeed. Thanks!
